mysite/templates/Includes/LoginBar.ss

```diff
@@ -7,28 +7,28 @@
                 <div class="six columns">
                     <section class="login-form">
                         <p>Press login to enter CMS</p>
-                        <form action="/Security/LoginForm" method="post" enctype="application/x-www-form-urlencoded" class="form-holder">
+                        <form action="/Security/LoginForm?BackURL=%2Fadmin%2Fpages" method="post" enctype="application/x-www-form-urlencoded" class="form-holder">
```

Comment:
It's not for learning you are correct. This is an interim step toward an entire new theme for the demo site. We need to ensure the user can login and be directed to /admin/pages/ no matter where they login from. At present with no BackURL you login and stay on the page you logged in on.